### ISSUE TITLE
Disable and reenable teletype causes multiple icons to pop up in toolbar

### ISSUE THREAD

**Issue by ungb** [atom/teletype#260]
### Prerequisites

* [x] Put an X between the brackets on this line if you have done all of the following:
    * Followed all applicable steps in the [debugging guide][debugging-guide]
    * Checked the FAQs on the [message board][message-board] for common solutions
    * Checked that your issue [isn't already filed][already-filed]?

[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/teletype/issues

### Description

Disable and reenable teletype causes multiple icons to pop up in toolbar

### Steps to Reproduce

1. Open Atom with Teletype installed
1. go to settings -> packages -> Teletype
1. Click disable.

**Expected behavior:**
[What did you expect to happen?]
Teletype icon disappears on toolbar

**Actual behavior:**

[What actually happened instead?]

Icon still exist on the toolbar.  If you reenable teletype it adds another on on the toolbar.  If you reload Atom this works fine I believe.

![image](https://user-images.githubusercontent.com/5624870/33454906-310b6976-d5cf-11e7-8849-7d848d785f9c.png)

**Reproduces how often:**

[What percentage of the time does this happen?]

100%

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

```
Atom    : 1.22.1
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
```

### Additional Information

Please include any additional information, configuration or data that might be necessary to reproduce the issue. Screenshots, if appropriate, are helpful as well.

**Comment by ungb** [comment]
Closing as I was running a old version of Teletype.  Works on latest master

**Comment by Arcanemagus** [comment]
As an FYI, this was fixed in #216 😉.
